uproot/interp/auto.py

```diff
@@ -200,7 +200,7 @@ def transform(node, tofloat=True):
                             numbits = None
                         else:
                             low, high, numbits = spec
-                    except:
+                    except Exception:
```

Comment:
We can be explicit about the classes caught by general exception handlers. (Is that in a style guide? I'll start doing it, if so.)

Reply:
I'm going to quote from PEP8 rather than writing the same thing less clearly:

> When catching exceptions, mention specific exceptions whenever possible instead of using a bare `except:` clause:
> ```python
> try:
>     import platform_specific_module
> except ImportError:
>    platform_specific_module = None
> ```
> A bare `except:` clause will catch `SystemExit` and `KeyboardInterrupt` exceptions, making it harder to interrupt a program with Control-C, and can disguise other problems. If you want to catch all exceptions that signal program errors, use `except Exception:` (bare except is equivalent to `except BaseException:`).
> 
> A good rule of thumb is to limit use of bare 'except' clauses to two cases:
> 
>    * If the exception handler will be printing out or logging the traceback; at least the user will be aware that an error has occurred.
>    * If the code needs to do some cleanup work, but then lets the exception propagate upwards with raise. try...finally can be a better way to handle this case.